pkg/processor/trigger/partitioned/kafka/partition.go

```diff
@@ -29,17 +31,38 @@ import (
 type partition struct {
 	*partitioned.AbstractPartition
 	partitionID       int
+	offset            int64
 	partitionConsumer sarama.PartitionConsumer
 	event             Event
+	stopChan          chan bool
 }
 
-func newPartition(parentLogger logger.Logger, kafkaTrigger *kafka, partitionID int) (*partition, error) {
+func newPartition(parentLogger logger.Logger, kafkaTrigger *kafka, partitionConfig functionconfig.Partition) (*partition, error) {
 	var err error
+
+	partitionID, err := strconv.Atoi(partitionConfig.ID)
+	if err != nil {
+		parentLogger.ErrorWith("Bad partition ID", "id", partitionConfig.ID, "err", err)
```

Comment:
We shouldn't log on errors, rather trust that the system will log the error for us